tests/sles4sap/netweaver_ascs.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's SLES4SAP openQA tests
+
+# G-Summary: Checks NetWeaver's ASCS installation as performed by sles4sap/nw_ascs_install
```

Comment:
G-* is for generated - you wrote this, no?

Reply:
Yes. I reviewed several modules looking for a commenting standard, and couldn't find it but bumped into the G-* and assumed they were used to generate documentation, not that they were generated. Will fix that as well.